packages/contracts/contracts/L1/ValidatorPool.sol

```diff
@@ -231,12 +231,7 @@ contract ValidatorPool is ReentrancyGuardUpgradeable, ISemver {
      */
     function deposit() external payable {
         require(
-            block.timestamp <
-                L2_ORACLE.computeL2Timestamp(
-                    L2_ORACLE.startingBlockNumber() +
-                        (TERMINATE_OUTPUT_INDEX + 1) *
-                        L2_ORACLE.SUBMISSION_INTERVAL()
-                ),
+            L2_ORACLE.nextOutputIndex() < TERMINATE_OUTPUT_INDEX + 1,
```

Comment:
I think the conflict resolving is incorrect, please check.

Reply:
This was a fix to reduce gas and enable the e2e testing. The timestamp of e2e actions is set to the current real-time timestamp, so that testing the transition from `ValidatorPool` to `ValidatorManager` fails at the initial stage.